scripts/core/editor3/html/index.js

```diff
@@ -4,11 +4,12 @@ import {HTMLParser} from './from-html';
 /**
  * @name toHTML
  * @param {Object} contentState
+ * @param {bool} withAnnotations
  * @description Converts DraftJS ContentState to HTML.
  * @returns {string} HTML
  */
-export function toHTML(contentState) {
-    return new HTMLGenerator(contentState).html();
+export function toHTML(contentState, withAnnotations) {
+    return new HTMLGenerator(contentState).html(withAnnotations);
```

Comment:
I'm thinking that from an API point of view it might be more friendly and would make more sense to add a second param for options to the `HTMLGenerator` so that it looks something like:
```js
return new HTMLGenerator(contentState, opts).html();
```
Just in case more stuff gets added later, and it might be needed earlier than the `html()` call.